packages/docs/src/components/DocsContent.vue

```diff
@@ -22,7 +22,7 @@
       <MarkdownView
         v-else-if="block.type === BlockType.TITLE"
         :key="block.type + index"
-        :value="`# ${$t(block.translationString)}`"
+        :value="`# ${$tie(block.translationString)}`"
```

Comment:
Do we have titles that don't have to be translated?

Reply:
I returned it as it was